picard/script.py

```diff
@@ -1047,10 +1055,12 @@ def func_map_multi(parser,
 
     Arguments:
         parser: The ScriptParser object used to parse the script.
-        multi: The multi-value to be iterated.
-        loop_code: String of script code to be processed on each iteration that yields the new value for
-            the multi-value element.
-        separator: String used to separate the elements in the multi-value.
+        multi: The ScriptVariable/Function that evaluates to a multi-value which
+            is to be iterated.
+        loop_code: The ScriptVariable/Function of script code to be processed on
+            each iteration that yields the new value for the multi-value element.
+        separator: A string or the ScriptVariable/Function that evaluates to the
+            string used to separate the elements in the multi-value.
```

Comment:
Changes related to documentation of previous functions should go in a separate PR.

Reply:
I backed them out of this PR and will create a new PR (later when the dust settles a little).  :-)